### PR TITLE
Add tilt and height to arc-layer

### PR DESCRIPTION
**Pull request by georgios-uber**

### CODE REVIEW

**Comment by georgios-uber** [comment]
Example usage:
screen shot 2019-03-05 at 9 27 08 pm

File: docs/layers/arc-layer.md [hunk 1]
```diff
@@ -123,6 +123,19 @@ The stroke width of each object, in meters.
...
+
+* Default: `1`
+
+How tall each arc is.
+
+##### `getTilt` (Function|Number, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square")
+
+* Default: `0`
+
+Use to tilt the arc to the side if you have multiple arcs with the same source and target positions.
```
Comment (Pessimistress) [inline docs/layers/arc-layer.md]: What is the unit of this value?

File: docs/layers/arc-layer.md [hunk 2]
```diff
@@ -123,6 +123,19 @@ The stroke width of each object, in meters.
 * If a number is provided, it is used as the stroke width for all objects.
 * If a function is provided, it is called on each object to retrieve its stroke width.
 
+##### `getHeight` (Function|Number, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square")
+
+* Default: `1`
+
+How tall each arc is.
```
Comment (Pessimistress) [inline docs/layers/arc-layer.md]: Clarify that it is the multiplier of height

**Comment by georgios-uber** [comment]
tilt

File: docs/layers/arc-layer.md (hunk 1 above)
Comment (Pessimistress) [reply docs/layers/arc-layer.md]: Call out that this is in degrees.

Comment (georgios-uber) [reply docs/layers/arc-layer.md]: Done

**Pull request merged**